src/ztc/api/serializers/besluittype.py

```diff
@@ -1,21 +1,24 @@
+from django.utils.translation import ugettext_lazy as _
+
 from rest_framework import serializers
+from rest_framework.exceptions import PermissionDenied
 
-from ...datamodel.models import BesluitType
+from ...datamodel.models import BesluitType, InformatieObjectType, ZaakType
 
 
 class BesluitTypeSerializer(serializers.HyperlinkedModelSerializer):
     informatieobjecttypes = serializers.HyperlinkedRelatedField(
         view_name='informatieobjecttype-detail',
         many=True,
         lookup_field='uuid',
-        read_only=True
+        queryset=InformatieObjectType.objects.all()
```

Comment:
Are you sure it doesn't show all `InformationObjectTypes` now, regardless whether it's part of the `Besluit` or not?

I think there needs to be a `source` attribute, like `source=informatieobjecttype_set` or something.